front50-core/src/main/groovy/com/netflix/spinnaker/front50/exception/ApplicationAlreadyExistsException.java

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.front50.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "Application already exists")
+public class ApplicationAlreadyExistsException extends RuntimeException {
```

Comment:
I know this was just a straight across port, but think its worth shimming this under SpinnakerException?

Reply:
(I guess the same comment for all the custom exceptions in here)